sonic-chassisd/scripts/chassisd

```diff
@@ -630,6 +881,61 @@ class ChassisdDaemon(daemon_base.DaemonBase):
         else:
             self.log_warning("Caught unhandled signal '{}' - ignoring...".format(SIGNALS_TO_NAMES_DICT[sig]))
 
+    def set_initial_dpu_admin_state(self):
+        """Send admin_state trigger once to modules those are powered up"""
+        lock = threading.Lock()
+        threads = []
+        for module_index in range(0, self.module_updater.num_modules):
```

Comment:
@rameshraghupathy I am not sure why we cannot handle all DPUs in one single thread.

Reply:
> @rameshraghupathy I am not sure why we cannot handle all DPUs in one single thread.
@prgeor It is not that we can't.  We can but the call goes sequentially to each DPU and power cycling is a time consuming process and each DPU requires roughly 3 mins. So to power cycle a chassis we need 24+ mins.  When we fork all 8 finishes in 3 mins